example/e2e/detoxTest.spec.js

```diff
@@ -33,7 +50,7 @@ describe('Example', () => {
         element(by.type('UIPickerView').withAncestor(by.id('dateTimePicker'))),
       ).toBeVisible();
     } else {
-      const testElement = await element(
```

Comment:
Why do you removed the `await`?

Reply:
I removed it because it's not supposed to be there :) `element` is not a async function so there is no need to await it

Reply:
I did not know that, there are more cases using `await` before `element` but I think this could be addressed in another PR